docs/deployment_ops_guide.md

```diff
@@ -0,0 +1,850 @@
+$ go tool pprof ./path/to/binary ./heap.pprof.out
+```
+
+Inspect profile interactively via `http://localhost:1234/debug/pprof` and host a web interface at `http://localhost:8000` (which opens automatically once profile is captured):
+
+```
+$ go tool pprof -http :8000 :1234/debug/pprof/heap
+```
+
+## Common Configuration Options
+
+(TODO: I'm expecting there to be overlap w regular Lily docs here. Leaving this for last.)
+
+The options described here are not exhaustive and focus on those most helpful to operator usage. Please refer to the Lily Documentation for full details on each option.
+
+### Configuration Precedence
+
+There are many ways to define certain options in Lily and precedence is defined in the following order:
+
+0. Command line flag value from user
+1. Environment variable (if specified)
+2. Configuration file (if specified)
+3. Default defined on the flag
+
+### Options: `lily init`
+
+#### option: Import Snapshot
+
+- arg: `lily init --import-snapshot`
+- env: `$LILY_SNAPSHOT`
+
+Import a snapshot to quickly bootstrap a new node syncing to a network. (This is especially useful for long-lived networks, such as `mainnet`.)
+
+Protocol Labs maintains a snapshot for `mainnet` at [https://docs.filecoin.io/get-started/lotus/chain/#syncing](https://docs.filecoin.io/get-started/lotus/chain/#syncing). (*Note: A lightweight snapshot will not contain complete historical state and will fail to work for historical indexing. Please double-check the snapshot chosen for import.*)
+
+Example:
+
+```
+$ lily init --import-snapshot="https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car"
+```
+
+#### option: Initialize Repo Path
+
+- arg: `lily init --repo`
+- env: `$LILY_REPO`
+- default: `$HOME/.lily`
+
+Create a new repo at a custom path for Lily's local state on the filesystem (also known as the `repo`). The repo contains chain/block data, keys, configuration, and other operational data. This folder is portable and can be relocated as needed.
+
+If a directory already exists at the location, it will be untouched by `lily init`.
+
+Example:
+
+```
+$ lily init --repo=/var/lib/lily
+```
+
+#### option: Initialize Config
+
+- arg: `lily init --config`
+- env: `$LILY_CONFIG`
+
+Create a new configuration template at the location specified. The file uses the TOML format. This file is portable and can be relocated as needed.
+
+If a configuration file already exists at the location, it will be untouched by `lily init`.
+
+Example:
+```$ lily init --config=/var/lib/lily/config.toml
+```
+
+### Args for `lily daemon`
+
+#### option: Daemon w custom Repo Path
+
+- arg: `lily daemon --repo`
+- env: `$LILY_REPO`
+
+Specify a custom path for Lily to read and manage its state on the filesystem (also known as the `repo`). The repo contains chain/block data, keys, configuration, and other operational data. This folder is portable and can be relocated as needed.
```

Comment:
Needs a big warning saying this is going to be huge, needs to live in a very fast disk etc.